src/Doxyfile.in

```diff
@@ -874,7 +868,7 @@ EXCLUDE_SYMBOLS        =
 # that contain example code fragments that are included (see the \include
 # command).
 
-EXAMPLE_PATH           = @CMAKE_SOURCE_DIR@/biblio/ \
+EXAMPLE_PATH           = @CMAKE_CURRENT_BINARY_DIR@ \
```

Comment:
Same here, except that there was a bug that should be fixed by saying:
```suggestion
EXAMPLE_PATH           = @CMAKE_SOURCE_DIR@ \
```

As we do:
```cpp
 * \verbinclude  biblio/how_to_cite_gudhi.bib
```